packages/components/data-table/src/data-table.styles.js

```diff
@@ -42,6 +42,11 @@ const Body = styled.tbody`
 const Row = styled.tr`
   display: contents;
   ${getClickableRowStyle}
+  :hover {
+    button[id='rowExpandCollapseButton'] {
```

Comment:
ids to the rescue once more! 😄

Reply:
FYI, I think in this case you could also import the `RowExpandCollapseButton`component defined in `cell.styles.js` and use it as the selector, [as seen here](https://emotion.sh/docs/styled#targeting-another-emotion-component). It would allow us to avoid setting an id, except you are also using the id for the unit test, so we can keep the id.

This comment is just for awareness and in case you wanna try it as an experiment for your CSS-in-JS skills 😄

Reply:
Using the approach from `emotions.sh` (the one that you mentioned) make much more sense. Also since the `RowExpandCollapseButton` has a label and we can use the `queryAllByLabelText` in the tests, to find all the expand/collapse CTAs and count them. 

These changes will make the usage of `id` on `RowExpandCollapseButton` absolute and therefore I will remove it in the next commit.

Reply:
Cool!